spec/lib/audit/checksum_spec.rb

```diff
@@ -28,30 +29,37 @@
     end
   end
 
-  describe ".validate_disk_all_endpoints" do
-    it 'calls validate_disk once per endpoint' do
-      described_class.validate_disk_all_endpoints('2018-02-05 21:37:23 UTC', 'MD5')
-      skip 'we should figure out what they are and test them'
+  describe ".validate_disk_all_dirs" do
+    let(:subject) { described_class.validate_disk_all_dirs(last_checked_b4_date) }
+
+    it 'calls validate_disk once per storage root' do
+      expect(described_class).to receive(:validate_disk).exactly(Settings.moab.storage_roots.count).times
+      subject
     end
 
     it 'calls validate_disk with the right arguments' do
-      described_class.validate_disk_all_endpoints('2018-02-05 21:37:23 UTC', 'MD5')
-      skip 'we should figure out what they are and test them'
+      Settings.moab.storage_roots.each do |storage_root|
+        expect(described_class).to receive(:validate_disk).with(
+          last_checked_b4_date,
+          "#{storage_root[1]}/#{Settings.moab.storage_trunk}"
+        )
+      end
+      subject
     end
```

Comment:
if we wanted to go nuts we could add tests looking for the start and end statements in the logs.  But I doubt it's worth it.